Source/carthage/CopyFrameworks.swift

```diff
@@ -39,7 +39,9 @@ public struct CopyFrameworksCommand: CommandProtocol {
 										.then(SignalProducer<(), CarthageError>.empty)
 								}
 						}
-				}
+					}
+					// Copy as many frameworks as possible in parallel.
+					.observe(on: QueueScheduler(name: "org.carthage.CarthageKit.CopyFrameworks.copy"))
```

Comment:
After measuring this again, it looks like `.merge` (with no scheduler) results in the same speedup as both observing or starting on a background scheduler. I'd assume that this is due to the fact that `Task` is non-blocking, and most of the work in `copy-framework` spent running `Task`s.

However, it does seem to be more correct to give each copied framework its own scheduler to ensure that if there ever is a blocking operation it doesn't result in another slowdown. Up to you!